### DISCUSSION TITLE
Simple but powerful Javascript SDK + Pre-built chat UI

### DISCUSSION THREAD

**Discussion by bestassist112** [Real-Time-Chat/react-chat-app#3] [Q&A] [answered]
The chat will seamlessly fit into your existing app as the Javascript SDK comes with a pre-built inbox, chatbox, and popup that you can customize and style easily with TalkJS Themes.

**Comment by VitaminC1211** [comment, marked as answer]
TalkJS is packed with out-of-the-box features. From push, SMS, and email notifications to emoji, message search, typing indicators, file sharing, etc. With support for some advanced features like email & SMS fallback, word blacklists, location sharing, and real-time message translations.
